release-notes-1.0.0.md

```diff
@@ -127,19 +127,19 @@ Java interoperability is a key feature in jBallerina that allows you to call Jav
 
 - Revamp the NATS connector to support both NATS and Streaming Servers.
 - Introduce the standard library module-wise errors as a replacement for the builtin `error`.
-  E.g., Ballerina HTTP Error types include `http:ClientError`, `http:ListenerError`, `http:ClientAuthError` etc.
+  e.g., Ballerina HTTP Error types include `http:ClientError`, `http:ListenerError`, `http:ClientAuthError` etc.
 - Introduce capability to engage custom providers and handlers for inbound/outbound authentication.
 - Introduce OAuth2 inbound authentication.
 - Introduce own modules for different authentication mechanisms (JWT, LDAP, OAuth2 etc.).
 - Introduce prior knowledge support to the HTTP/2 client.
 - Add flow control support to HTTP/2 client and server.
 - Introduce XSLT transformation support.
-- `h2` and `mysql` database client modules and the `sql` module have been discontinued. The `ballerinax/java.jdbc` client module can be used to interact with relational databases.
-- The ByteChannel read API was updated to return only `byte[]|io:Error`, removing the previously returned read byte length.
+- `ballerina/h2` and `ballerina/mysql` database client modules and the `ballerina/sql` module have been discontinued. The `ballerinax/java.jdbc` client module can be used to interact with relational databases.
+- The `ByteChannel` read API was updated to return only `byte[]|io:Error`, removing the previously returned read byte length.
```

Comment:
This sentence still sounds weird. Any ideas on how this can be improved? @MaryamZi

Reply:
```suggestion
- The byte channel read API was updated to return only `byte[]|io:Error`.
```